packages/vue/src/components/atoms/SfColor/SfColor.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <button
+    class="sf-color"
+    :class="{ 'sf-color--active': isActive }"
+    :style="style"
+    :aria-pressed="isActive"
```

Comment:
```suggestion
    :aria-pressed="isActive.toString()"
```
if you set false than this props is removed :)